static/app/views/settings/projectDebugFiles/index.tsx

```diff
@@ -180,25 +179,23 @@ class ProjectDebugSymbols extends AsyncView<Props, State> {
           `)}
         </TextBlock>
 
-        {features.includes('symbol-sources') && (
-          <Fragment>
-            <PermissionAlert />
-            <ExternalSources
-              api={this.api}
-              location={location}
-              router={router}
-              projSlug={project.slug}
-              organization={organization}
-              customRepositories={
-                (project.symbolSources
-                  ? JSON.parse(project.symbolSources)
-                  : []) as CustomRepo[]
-              }
-              builtinSymbolSources={project.builtinSymbolSources ?? []}
-              builtinSymbolSourceOptions={builtinSymbolSources ?? []}
-            />
-          </Fragment>
-        )}
+        <PermissionAlert />
```

Comment:
I think you should leave the feature check for `symbol-sources`, i.e. hide the entire UI if the `symbol-sources` feature is not available.